doc/man3/SSL_alloc_buffers.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+SSL_free_buffers, SSL_alloc_buffers - manage SSL structure buffers
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_free_buffers(SSL *ssl);
+ int SSL_alloc_buffers(SSL *ssl);
+
+=head1 DESCRIPTION
+
+SSL_free_buffers() frees the read and write buffers of the given B<ssl>.
+SSL_alloc_buffers() allocates the read and write buffers of the given B<ssl>.
+Using these functions can save around 34k per SSL connection.
+
+The SSL_MODE_RELEASE_BUFFERS mode releases read or write buffers when the
+buffers have been drained. If this is too aggressive, the buffers can be
+freed and allocated using these functions.
+
+After freeing the buffers, the buffers are automatically reallocted upon a
+new read or write. The SSL_alloc_buffers() does not need to be called, but
+can be used to control when the buffers are allocated.
```

Comment:
That control is rather weak, isn't it?  I think I'd rather have something along the lines of "but can be used to make sure the buffers are pre-allocated" with some words about why that would be desirable...